scripts/copy-code-to-clipboard.js

```diff
@@ -1,10 +1,20 @@
 const ClipboardJS = require('clipboard')
-const clipboard = new ClipboardJS('.btn-clipboard')
-
-clipboard.on('success', (e) => {
-  e.clearSelection()
+const clipboard = new ClipboardJS('.btn-clipboard', {
+  // strip leading console $ from copied codeblocks
+  text: trigger => {
+    const codeBlockID = trigger.getAttribute('data-clipboard-target').substr(1)
+    let codeBlock = document.getElementById(codeBlockID).textContent
+    console.log(codeBlock)
+    codeBlock = codeBlock.replace(/\n\$/g, '\n') // eslint-disable-line no-useless-escape
+    if (codeBlock.startsWith('$ ')) {
+      codeBlock = codeBlock.substr(2)
+    }
```

Comment:
How do you feel about changing this whole section to something like

```javascript
codeBlock = codeBlock.replace(/^\$\W*/gm, '')
```

This handles any number of spaces after the `$` and doesn't require the `if` statement to take care of the first line.